### DISCUSSION TITLE
VS Code: how to change literal variables to other data types

### DISCUSSION THREAD

**Discussion by learntocode97** [microsoft/vscode-discussions#193] [Extension Development QnA] [answered]
Hi, I'm new to VSC. After installing VSC and Python extensions. I created a .py  file, when I type in `test =12` and move my mouse to "test", it shows (variable) test: Literal[12]. I also try with other data types like string, boolean, and integer, but it's always the same. Only float is shown correctly. 
I tried to google about `option infer off` and `option strict on`, but I can't find a way to run it.  
How can VSC show my data type automatically correctly?
Many thanks in advance.

**Comment by gjsjohnmurray** [comment, marked as answer]
Please go to https://github.com/microsoft/vscode-python, search for duplicates, then open a new one if necessary. This discussion platform is currently focused on developers who are **building** extension for VS Code.
